wrappers/python/examples/wnt/README.md

```diff
@@ -0,0 +1,64 @@
+# WNT API Examples
+
+This folder contains the WNT API Examples.
+
+## Installation
+
+The examples require that the wirepas_messaging packet is installed. Please see the README.md
```

Comment:
I would change the text to 

Before running the examples please ensure that the [wirepas_messaging *package* is installed][link_to_readme].

Reply:
> 
> 
> I would change the text to
> 
> Before running the examples please ensure that the [_wirepas_messaging package_ is installed](https://github.com/wirepas/backend-apis/tree/master/wrappers/python).
> 
> Please use full urls for the link:



> 
> 
> I would change the text to
> 
> Before running the examples please ensure that the [_wirepas_messaging package_ is installed](https://github.com/wirepas/backend-apis/tree/master/wrappers/python).
> 
> Please use full urls for the link:

Done